specification/cognitiveservices/data-plane/ContentModerator/readme.md

```diff
@@ -37,32 +37,4 @@ csharp:
 ``` yaml
 directive:
   - suppress: R2022                        # the message to suppress
-```
-
-## Python
```

Comment:
Why is section removed for Python?

Reply:
Also I noticed that in the python setting , namespace  is "azure.cognitiveservices.vision.contentmoderator" and output folder is "$(python-sdks-folder)/azure-cognitiveservices-vision-contentmoderator/azure/cognitiveservices/vision/contentmoderator". As content moderator is an independent cognitive service and is not under vision service, will it be a problem if I modify the namespace to "azure.cognitiveservices.contentmoderator" and folder to "$(python-sdks-folder)/azure-cognitiveservices-vision-contentmoderator/azure/cognitiveservices/contentmoderator"  ?

Reply:
For folder, you mean $(python-sdks-folder)/azure-cognitiveservices-contentmoderator/azure/cognitiveservices/contentmoderator?

Reply:
Yes

Reply:
It's up to you since it's your service, as long as it matches the business logic - from SDK's perspective either is valid and we can generate functional code.

This seems to generate just fine: https://github.com/v-sodsou/azure-rest-api-specs/blob/16b0bd0bcb7d79ad83d2372cd751eeab307c0b65/specification/cognitiveservices/data-plane/EntitySearch/readme.md

Reply:
Ok.Thanks! I have modified the read me file accordingly. Could you please review it.